Tests/TuistLoaderTests/Models+ManifestMappers/TargetScript+ManifestMapperTests.swift

```diff
@@ -33,24 +33,97 @@ final class TargetScriptManifestMapperTests: TuistUnitTestCase {
         // Given
         let temporaryPath = try temporaryPath()
         let generatorPaths = GeneratorPaths(manifestDirectory: temporaryPath)
+        try createFiles([
+            "foo/bar/a.swift",
+            "foo/bar/b.swift",
+            "foo/bar/aTests.swift",
+            "foo/bar/bTests.swift",
+            "foo/bar/kTests.kt",
+            "foo/bar/c/c.swift",
+            "foo/bar/c/cTests.swift",
+        ])
+
+        let manifest = ProjectDescription.TargetScript.test(
+            name: "MyScript",
+            tool: "my_tool",
+            order: .pre,
+            arguments: ["arg1", "arg2"],
+            inputPaths: ["foo/bar/**/*.swift"],
```

Comment:
I was not planning to touch this part but It seems this test was checking the string equivalent. After changing it to a glob, I started resolving the glob earlier which means that it's actually looking for the folders/files. So, I followed the similar approach from `TargetTests`